internal/config/state.go

```diff
@@ -99,12 +99,21 @@ type LibraryState struct {
 	// If not set, this defaults to the `source_roots`.
 	// A more specific `preserve_regex` takes precedence.
 	RemoveRegex []string `yaml:"remove_regex" json:"remove_regex"`
+	// Path of commits to be excluded from parsing while calculating library changes.
+	// If all files from commit belong to one of the paths it will be skipped.
+	ReleaseExcludePaths []string `yaml:"release_exclude_paths,omitempty" json:"release_exclude_paths,omitempty"`
```

Comment:
Please can you share an example of how this will be used? Historically we would rely on the conventional commit message to decide if a release is needed.